src/slashcommands/Help.ts

```diff
@@ -18,16 +19,22 @@ export class Help implements SlashCommand {
 	options = [];
 	requiredPermissions: bigint[] = [
 		Permissions.FLAGS.SEND_MESSAGES,
-		Permissions.FLAGS.EMBED_LINKS,
+		Permissions.FLAGS.EMBED_LINKS, 
 		Permissions.FLAGS.MANAGE_MESSAGES,
 		Permissions.FLAGS.ADD_REACTIONS,
 	];
 	async run(bot: Bot, interaction: CommandInteraction): Promise<void> {
 		try {
+			type cmdList = {[index:string]: string};
+			let cmds = {} as cmdList;
+			bot.slashCommands.forEach((command)=>{
+				cmds[command.name] = command.description;
+				console.log(`${command.name}: ${command.description}`);
```

Comment:
I will never merge a pr that contains console.log

Reply:
xD